datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -1397,6 +1404,56 @@ type Dataset implements EntityWithRelationships & Entity & BrowsableEntity {
     exists: Boolean
 }
 
+type ExternalRoleAssociation {
+
+    """
+    The extRole itself
+    """
+    externalRoleMetadata: ExternalRoleMetadata!
+
+    """
+    Reference back to the tagged urn for tracking purposes e.g. when sibling nodes are merged together
+    """
+    associatedUrn: String!
+
+}
+
+type ExternalRolesMetadata {
+    roles: [ExternalRoleAssociation!]
+}
+
+type ExternalRoleMetadata implements Entity{
+    """
+    A primary key of the Metadata Entity
+    """
+    urn: String!
+
+    """
+    A standard Entity Type
+    """
+    type: EntityType!
+
+    """
+    List of relationships between the source Entity and some destination entities with a given types
+    """
+    relationships(input: RelationshipsInput!): EntityRelationshipsResult
+
+    id: String!
+
+    properties: ExternalRoleProperties!
+
+    provisionedUsers: [CorpUser!]
```

Comment:
Let's put this inside a wrapper object... Called "users"

Reply:
Or better yet - Actors

Reply:
Done